BrainPortal/app/views/tasks/show.html.erb

```diff
@@ -57,6 +57,26 @@
       <% confirm += "\n\nIMPORTANT NOTE: This will also remove its archived work directory." if @task.workdir_archive %>
       <%= link_to 'Remove Task',  { :action => 'operation', :operation => 'delete', 'tasklist[]' => @task.id }, :confirm => confirm, :class  => "button menu_button", :method => :post %>
     <% end %>
+    <% if @task.status =~ /Failed*|Completed/ %>
```

Comment:
Don't use state names explicitely; use the contants defined in cbrain_task.rb ( `COMPLETED_STATUS`, `FAILED_STATUS` etc). Yes, other buttons were already making such comparisons, let's clean them up too.